styles/styles.css

```diff
@@ -18,14 +18,25 @@
   --primary-color: #552448;
   --primary-color-lighten: rgba(98 98 121 / 12.5%);
   --body-color: #3a3a3a;
-  --lightgray: #f6f6f6;
+  --lightgrey: #f6f6f6;
+  --darkgrey: #797676;
   --black: #000;
-  --gray: #d8d8d8;
+  --grey: #d8d8d8;
   --white: #fff;
+  --greyblue: #36758d;
   --white-background: #fff;
-  --border-light-gray: #d8d8d8;
-  --border-medium-gray: #aaa;
+  --border-light-grey: #d8d8d8;
+  --border-medium-grey: #aaa;
   --primary-color-svg-filter: invert(14%) sepia(47%) saturate(1309%) hue-rotate(272deg) brightness(96%) contrast(89%);
+  --seller-advice: rgb(240 240 224);
```

Comment:
Are these used anywhere besides blog? Can they be referenced from a blog css and imported into the appropriate sub-blocks?

Reply:
I'm still determining. maybe we can do the second round of CSS clean up at the end of the project?

Reply:
Sounds good.